src/dispensing-tiles/dispensing-tiles.resource.tsx

```diff
@@ -0,0 +1,33 @@
+import useSWR from "swr";
+import useSWRImmutable from "swr/immutable";
+import { FetchResponse, openmrsFetch } from "@openmrs/esm-framework";
+
+export function useMetrics() {
+  const metrics = {
+    orders: 43,
+    orders_for_home_delivery: 4,
+    missed_collections: 12,
+  };
+  const { data, error } = useSWR<{ data: { results: {} } }, Error>(
+    `/ws/rest/v1/queue?`,
+    openmrsFetch
```

Comment:
is this queue endpoint and the services below copied from elsewhere and not applicable here?

Reply:
This was just for testing the api fetch with another endpoint.